data/tools/apps.yml

```diff
@@ -92,6 +92,18 @@ apps:
     -   func
     description: BIDS Application for the Configurable Pipeline for the Analysis of Connectomes (C-PAC)
 
+-   gh: ln2t/cvrmap
```

Comment:
Do you want it capitalized? Currently [looks like](https://bids-website--637.org.readthedocs.build/en/637/tools/bids-apps.html):

![image](https://github.com/user-attachments/assets/c99be478-5d58-4889-92cd-0e39df6ec463)


```suggestion
-   gh: ln2t/CVRmap
```